package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "node_web_app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha",
+    "start": "node bootstrap.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brsutton/node_web_app.git"
+  },
+  "author": "Brian Sutton",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/brsutton/node_web_app/issues"
+  },
+  "homepage": "https://github.com/brsutton/node_web_app#readme",
+  "dependencies": {
+    "aws-sdk": "^2.172.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "bcrypt": "^1.0.3",
+    "hapi": "^17.1.1",
+    "hapi-auth-basic": "^5.0.0",
+    "mocha": "^4.0.1"
```

Comment:
please move appropriate things to dev dependencies as we don't need all of them for production code